BackEndApp/v1/views.py

```diff
@@ -24,7 +24,9 @@
     decrypt,
     get_git_username,
 )
+import logging
 
+logger = logging.get_logger(__name__)
```

Comment:
```suggestion
logger = logging.getLogger(__name__)
```

Reply:
Also could add a screenshot of how the logs look. I'm unable to view any of the logs, not sure if it's the logger configuration mistake

Reply:
Nevermind probably some django configuration cache.

Reply:
<img width="480" alt="Screenshot 2021-10-06 at 9 25 57 AM" src="https://user-images.githubusercontent.com/16982169/136138289-9ccfdf28-9c43-4450-8a57-69a519dd0482.png">